### DISCUSSION TITLE
Is it possible to install just the `scheduler` package?

### DISCUSSION THREAD

**Discussion by aodj-kara** [spulec/moto#7250] [Q&A] [answered]
Looking at the `setup.cfg` it doesn't appear to be mentioned, so I'm wondering if it's supporter, or if it needs to be exposed so that users can do `pip install moto[scheduler]` in the same way as is done for ec2/s3

**Comment by bblommers** [comment, marked as answer]
Hi @aodj-kara, `pip install moto[service]` only install any additional dependencies for that service. It will always install the entire Moto-package.

Considering the `scheduler` does not have any additional dependencies, `pip install moto` would be equivalent to `pip install moto[scheduler]`. 

The latter does throw a warning at the moment, because `scheduler` is not defined in `setup.cfg` - I'll make a note to still add it to the list to make sure the warning does not appear.

**Comment by aodj-kara** [comment]
Ah my mistake! Thanks for the explanation though ❤️
